<launch>
    <remap to="/camera_array/cam0/image_raw" from="/image_sender_0"/>
    <arg name="viz_output_path" default="/home/yang/test2.avi" />
    <group ns="vehicle">

        <node name="run_3cam_concat" pkg="mkz_intel" type="sync_3cams.py" output="screen"> </node>
        <node name="record_visualizations" pkg="image_view" type="video_recorder" output="screen">
            <remap to="sync_3cam" from="image"/>
            <param name="filename" value="$(arg viz_output_path)" />
            <param name="codec" value="X264" />
        </node>
    </group>


</launch>